LetsDoStuff.WebApi/Controllers/UserController.cs

```diff
@@ -22,6 +23,27 @@ public UserController(IUserService userService, ILogger<IUserService> logger)
             _logger = logger;
         }
 
+        /// <summary>
+        /// Get list of all users. To access this you need to have the "Admin" role.
+        /// </summary>
+        /// <returns>All users.</returns>
+        [Authorize]
+        [HttpGet("users")]
```

Comment:
Let's use controllers common route. Check activities and read more about rest ideology